web/templates/twig-source/incuna-surveys/form/integer-field.html

```diff
@@ -0,0 +1,14 @@
+{% block input %}
+    <input 
+        class="integer-input"
+        id="{% raw %}{{ id }}{% endraw %}"
+        name="{% raw %}{{ id }}{% endraw %}"
+        type="text" ng-model="model"
+        ensure-integer
+    >
+{% endblock %}
+{% block errors %}
+    <div class="error-block field-error ng-scope" ng-show="form[id].$invalid">
+        <p class="error" ><span class="ng-scope">A valid integer is required.</span></p>
```

Comment:
translate.
Why is `ng-scope` here? I thought it was added by angular when a directive was creating a new scope?